.github/workflows/backport-command.yml

```diff
@@ -70,11 +116,178 @@ jobs:
           ORIG_ASSIGNEES: ${{ steps.assignees.outputs.assignees }}
           ORIG_ISSUE_URL: ${{ github.event.client_payload.github.payload.issue.html_url }}
         run: |
-          gh issue create --title "[${BACKPORT_BRANCH}] ${ORIG_TITLE}" \
+          backport_issue_url=$(gh issue list --repo "${TARGET_ORG}/${TARGET_REPO}" \
+              --state open \
+              --search "[${BACKPORT_BRANCH}] ${ORIG_TITLE} in:title" \
+              --json url \
+              --milestone "${TARGET_MILESTONE}" \
+              --jq '.[0].url')
+          if [[ "$backport_issue_url" == "" ]]; then
+            gh issue create --title "[${BACKPORT_BRANCH}] ${ORIG_TITLE}" \
+              --label "kind/backport" \
+              --repo "${TARGET_ORG}/${TARGET_REPO}" \
+              --assignee "${ORIG_ASSIGNEES}" \
+              --milestone "${TARGET_MILESTONE}" \
+              --body "Backport ${ORIG_ISSUE_URL} to branch ${BACKPORT_BRANCH}"
+          else
+            gh issue comment "$ORIG_ISSUE_URL" --body "Backport issue already exists: $backport_issue_url"
+          fi
+
+      - name: Get reviewers of PR
+        if: needs.backport-type.outputs.commented_on == 'pr'
+        env:
+          REVIEWERS: ${{ toJson(github.event.client_payload.pull_request.requested_reviewers) }}
+        id: reviewers
+        run: echo ::set-output name=reviewers::$(echo "$REVIEWERS" | jq -r '.[].login' | paste -s -d ',' -)
+
+      - name: Get commits of PR
+        if: needs.backport-type.outputs.commented_on == 'pr'
+        env:
+          BACKPORT_PR_NUMBER: ${{ github.event.client_payload.pull_request.number }}
+          GITHUB_TOKEN: ${{ secrets.ACTIONS_BOT_TOKEN }}
+        id: backport_commits
+        run: |
+          backport_commits=$(gh api "repos/$TARGET_ORG/$TARGET_REPO/pulls/$BACKPORT_PR_NUMBER/commits" --jq .[].sha | paste -s -d ' ' -)
+          echo ::set-output name=backport_commits::$backport_commits
+
+      - uses: actions/checkout@v3
+        if: needs.backport-type.outputs.commented_on == 'pr'
+        with:
+          repository: ${{ steps.user.outputs.username }}/${{ steps.user.outputs.repo }}
+          token: ${{ secrets.ACTIONS_BOT_TOKEN }}
+
+      - name: Backport commits and get details
+        if: needs.backport-type.outputs.commented_on == 'pr'
+        env:
+          PR_NUMBER: ${{ github.event.client_payload.pull_request.number }}
+          GITHUB_TOKEN: ${{ secrets.ACTIONS_BOT_TOKEN }}
+          BACKPORT_BRANCH: ${{ steps.branch.outputs.branch }}
+          ORIG_TITLE: ${{ github.event.client_payload.github.payload.issue.title }}
+          BACKPORT_COMMITS: ${{ steps.backport_commits.outputs.backport_commits }}
+          IS_MERGED: ${{ github.event.client_payload.pull_request.merged }}
+        id: pr_details
+        run: |
```

Comment:
should also check if the PR to backport is a PR against `dev` branch.

Reply:
done